server/src/routes/api/v1/parkReviewsRouter.js

```diff
@@ -34,4 +34,36 @@ parkReviewsRouter.post("/", async (req, res) => {
   }
 });
 
+parkReviewsRouter.post("/:reviewId", async (req, res) => {
+  try {
+    const reviewId = req.params.reviewId;
+    const parkId = req.params.parkId;
+    await Review.query().deleteById(reviewId);
```

Comment:
this is nice, but you can actually use your relationMapping association to review the associated park BEFORE you delete the review, then delete the review, and this logic should be in a normal `reviews router`